input/docs/guidelines/framework/dispose-your-subscriptions.md

```diff
@@ -1,6 +1,6 @@
 # Dispose your subscriptions
 
-https://www.introtorx.com/Content/v1.0.10621.0/03_LifetimeManagement.html
+[comment]: <> (https://www.introtorx.com/Content/v1.0.10621.0/03_LifetimeManagement.html Broken link)
```

Comment:
http://introtorx.com/Content/v1.0.10621.0/03_LifetimeManagement.html#LifetimeManagement

Doesn't seem to support https is the problem

Reply:
Thanks, I will correct